MODULE.bazel

```diff
@@ -16,6 +16,7 @@ bazel_dep(name = "rules_cc", version = "0.1.1")
 bazel_dep(name = "rules_license", version = "1.0.0")
 bazel_dep(name = "rules_shell", version = "0.3.0")
 bazel_dep(name = "apple_support", version = "1.17.1", repo_name = "build_bazel_apple_support")
+bazel_dep(name = "aspect_bazel_lib", version = "2.19.4")
```

Comment:
Do we need to pull in this dependency? Could we instead add the attribute directly?

Reply:
We could vendor this whole file if we preferred: https://github.com/bazel-contrib/bazel-lib/blob/main/lib/resource_sets.bzl - I don't have a strong preference, WDYT?

Reply:
We've already got a very similar file I think we should use instead:
https://github.com/bazelbuild/rules_rust/blob/7d96cde89db4f41df3f464bf7fdd4e4c2d1a6c05/rust/private/rustc_resource_set.bzl

(some refactoring may be required)